tests/e2e/cairo1/test_cairo1_testing.py

```diff
@@ -22,10 +22,9 @@ def test_no_tests_found(protostar: ProtostarFixture):
 
 def test_failing_tests(protostar: ProtostarFixture, copy_fixture: CopyFixture):
     copy_fixture("cairo1_project", "./cairo1_project")
-    template_test_files = Path("cairo1_project/tests").glob("**/*")
-    for template_test_file in template_test_files:
-        template_test_file.unlink()
-    copy_fixture("cairo1/failing_test.cairo", "./cairo1_project/tests")
+    copy_fixture(
+        "cairo1/failing_test.cairo", "./cairo1_project/tests/failing_test.cairo"
+    )
```

Comment:
How does this work without removing existing test files? Below, we explicitly check that this line is in the output: `"Collected 1 suite, and 3 test cases"`

Reply:
I removed the default test from the template in this PR, so adding tests is now explicit. It's much better to add what you want IMHO than remove all tests (and also it caused some errors for me in case of nested directories god knows why)

Reply:
I see now, right, that totally makes sense! Nice.